spotbugs/etc/messages.xml

```diff
@@ -3283,6 +3291,66 @@ equal objects have equal hashCodes.
 and thus doesn't fulfill the requirement that equal objects have equal hashCodes.
 An instance of this class is used in a hash data structure, making the need to
 fix this problem of highest importance.
+]]>
+    </Details>
+  </BugPattern>
+  <BugPattern type="HE_NO_EQUALS_UNLIKE_PARENT_OR_OUTER_CLASS_OR_INTERFACE">
+    <ShortDescription>There is no equals() unlike its parent or outer class or interface.</ShortDescription>
+    <LongDescription>There is no equals() unlike its parent or outer class or interface.</LongDescription>
+    <Details>
+<![CDATA[
+  <p>No equals() unlike its parent or outer class or interface.</p>
```

Comment:
Does this take into account the possibility of using Apache Commons classes like `EqualsBuilder.reflectionEquals`?

Also, this needs more detail, not just a single short sentence.